src/App.js

```diff
@@ -1,24 +1,29 @@
-import React from 'react'
-import { BrowserRouter, Route } from 'react-router-dom'
+import React, { Suspense } from 'react'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 
-import './App.scss'
+import { PATH_URL } from 'common/constants'
 import ImgGallery from 'common/components/img-gallery/'
+import Preloader from 'ui-kit/preloader/preloader'
```

Comment:
I'd expect to have this kind of import:
```
import { Preloader } from 'ui-kit'
```
can you please make this change now and in the future as well ? 
whenever you add a component in **ui-kit**, imports should always look cleaner as the one from above